tljh/installer.py

```diff
@@ -170,14 +170,13 @@ def ensure_jupyterlab_extensions():
     Install the JupyterLab extensions we want.
     """
     extensions = [
-        '@jupyterlab/hub-extension',
```

Comment:
Why are you removing this extension?

Reply:
This plugin has been integrated to Juypterhub proper since 1.0.0: https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html

>JupyterHub Integration
We now include the JupyterHub extension in core JupyterLab, so you no longer need to install @jupyterlab/hub-extension. (#6451, #6428)

Reply:
Aha! I didn't know that. Every day is a school day :)